src/viz/validation.ts

```diff
@@ -50,19 +50,19 @@ No files have been created.
 };
 
 export const parseBucketName = (bucketPath: string): string | undefined => {
-  const matches = bucketPath.match(/(gs:\/\/[^/\s]+)/);
+  const matches = bucketPath.match(/(gs:\/\/[-a-zA-Z]+)\/?([-a-zA-Z]+\/?)*$/);
```

Comment:
Regarding this part of the regular expression:

    (gs:\/\/[-a-zA-Z]+)\/?

My guess is that your intention was to make the `gs://bucket/` prefix optional; however, all you've done is make the `/` optional. Move the `\/` to the inside of the group that contains `gs`.